doc/_utils/redirects.py

```diff
@@ -395,17 +395,28 @@
-    ("ug_zigbee_memory", "protocols/zigbee/memory"), # Zigbee memory requirements
-    ("ug_zigbee_configuring", "protocols/zigbee/configuring"), # Configuring Zigbee in nRF Connect SDK
-    ("ug_zigbee_configuring_libraries", "protocols/zigbee/configuring_libraries"), # Configuring Zigbee libraries in nRF Connect SDK
-    ("ug_zigbee_configuring_zboss_traces", "protocols/zigbee/configuring_zboss_traces"), # Configuring ZBOSS traces in nRF Connect SDK
-    ("ug_zigbee_adding_clusters", "protocols/zigbee/adding_clusters"), # Adding ZCL clusters to application (removed after 2.5.0)
-    ("ug_zigbee_other_ecosystems", "protocols/zigbee/other_ecosystems"), # Configuring Zigbee samples for other ecosystems
-    ("ug_zigbee_tools", "protocols/zigbee/tools"), # Zigbee tools
+    ("ug_zigbee_qsg", "protocols/zigbee/index"), # Zigbee quick start guide (removed after v3.0.0)
+    ("protocols/zigbee/qsg", "protocols/zigbee/index"),
+    ("ug_zigbee_supported_features", "protocols/zigbee/index"), # Supported Zigbee features (removed after v3.0.0)
+    ("protocols/zigbee/supported_features", "protocols/zigbee/index"),
+    ("ug_zigbee_architectures", "protocols/zigbee/index"), # Zigbee architectures (removed after v3.0.0)
+    ("protocols/zigbee/architectures", "protocols/zigbee/index"),
+    ("ug_zigbee_commissioning", "protocols/zigbee/index"), # Zigbee commissioning (removed after v3.0.0)
+    ("protocols/zigbee/commissioning", "protocols/zigbee/index"),
+    ("ug_zigbee_memory", "protocols/zigbee/index"), # Zigbee memory requirements (removed after v3.0.0)
+    ("protocols/zigbee/memory", "protocols/zigbee/index"),
+    ("ug_zigbee_configuring", "protocols/zigbee/index"), # Configuring Zigbee in nRF Connect SDK (removed after v3.0.0)
+    ("protocols/zigbee/configuring", "protocols/zigbee/index"),
+    ("ug_zigbee_configuring_libraries", "protocols/zigbee/index"), # Configuring Zigbee libraries in nRF Connect SDK (removed after v3.0.0)
+    ("protocols/zigbee/configuring_libraries", "protocols/zigbee/index"),
+    ("ug_zigbee_configuring_zboss_traces", "protocols/zigbee/index"), # Configuring ZBOSS traces in nRF Connect SDK (removed after v3.0.0)
+    ("protocols/zigbee/configuring_zboss_traces", "protocols/zigbee/index"),
+    ("ug_zigbee_adding_clusters", "protocols/zigbee/index"), # Adding ZCL clusters to application (removed after 2.5.0)
+    ("protocols/zigbee/adding_clusters", "protocols/zigbee/index"),
+    ("ug_zigbee_other_ecosystems", "protocols/zigbee/index"), # Configuring Zigbee samples for other ecosystems (removed after v3.0.0)
+    ("protocols/zigbee/other_ecosystems", "protocols/zigbee/index"),
+    ("ug_zigbee_tools", "protocols/zigbee/index"), # Zigbee tools (removed after v3.0.0)
+    ("protocols/zigbee/tools", "protocols/zigbee/index"),
     ("applications/nrf5340_audio/README", "applications/nrf5340_audio/index"), # nRF5340 Audio applications (landing)
     ("samples/samples_bl", "samples/bl"), # Bluetooth samples (landing)
```

Comment:
The tag file would also require updates - https://github.com/nrfconnect/sdk-nrf/blob/main/doc/_zoomin/ncs.tags.yml